### PR TITLE
Removed the 60 chars limitation and modified the error message format

### PR DESCRIPTION
**Pull request by rm75**
Before the change:

(venv) ECSDevKit:~/python-ecsclient # python
Python 2.7.9 (default, Dec 21 2014, 11:02:59) [GCC] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> from ecsclient.client import Client
>>> client = Client('3',
...                 username='root',
...                 password='test',
...                 token_endpoint='https://10.247.142.181:4443/login',
...                 ecs_endpoint='https://10.247.142.181:4443')
>>> print(client.user_info.whoami())
No handlers could be found for logger "ecsclient.common.token_request"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "ecsclient/common/other/user_info.py", line 36, in whoami
    return self.conn.get('user/whoami')
  File "ecsclient/baseclient.py", line 117, in get
    return self._request(url, params=params)
  File "ecsclient/baseclient.py", line 163, in _request
    headers=self._fetch_headers(),
  File "ecsclient/baseclient.py", line 106, in _fetch_headers
    token = self.token if self.token else self._token_request.get_token()
  File "ecsclient/common/token_request.py", line 106, in get_token
    return self.get_new_token()
  File "ecsclient/common/token_request.py", line 75, in get_new_token
    raise ECSClientException.from_response(req, message=msg)
ecsclient.common.exceptions.ECSClientException: Invalid username or password - Access is denied due to invalid or expired credentials - Unauthorized: https://10.247.142.181:4443/login 401 Unauthorized  [first 60 chars of response] {"code":401,"description":"Unauthorized","details":"Access i

After the change:
------------------
>>> from ecsclient.client import Client
>>> client = Client('3',
...                 username='root',
...                 password='test',
...                 token_endpoint='https://10.247.142.181:4443/login',
...                 ecs_endpoint='https://10.247.142.181:4443')
>>> print(client.user_info.whoami())
No handlers could be found for logger "ecsclient.common.token_request"
ecsclient.common.exceptions.ECSClientException:

Error Message: Invalid username or password
ECS_details: Access is denied due to invalid or expired credentials
ECS_description: Unauthorized

HTTP/S Request/Response Details:
-------------------------------
Scheme: https
Host: 10.247.142.181
Port:4443
Path: /login
Response_Status: 401
Response_Reason: Unauthorized
Response_content: {"code":401,"description":"Unauthorized","details":"Access is denied due to invalid or expired credentials"}

>>> client = Client('3',
...                 username='root',
...                 password='test',
...                 ecs_endpoint='https://10.247.142.181:4443')
Traceback (most recent call last):
  File "<stdin>", line 4, in <module>
  File "ecsclient/client.py", line 32, in Client
    return client_class(*args, **kwargs)
  File "ecsclient/v3/client.py", line 25, in __init__
    super(Client, self).__init__(*args, **kwargs)
  File "ecsclient/baseclient.py", line 54, in __init__
    raise ECSClientException("'token_endpoint' not provided and missing 'token'|'token_path'")
ecsclient.common.exceptions.ECSClientException:

Error Message: 'token_endpoint' not provided and missing 'token'|'token_path'

### CODE REVIEW

**Comment by keedya** [comment]
👍

**Comment by keedya** [comment]
@twincitiesguy This PR looks good; Are we clear to merge it?

**Comment by twincitiesguy** [comment]
I think we still need to limit the output form the response body.  What if there is some bug, and the response actually contains 1GB (maybe it's object data)?  Maybe instead of 60 chars, we limit to 8000 chars?  Or some value that will reasonably include error details, but will not flood the log if something crazy happens

**Comment by padthaitofuhot** [comment]
This PR addresses issue #62

**Comment by padthaitofuhot** [comment]
I agree with @twincitiesguy here.  Limiting exception data to something reasonable makes sense.  60 chars was obviously too short, but 4k or 8k makes sense to me.

File: ecsclient/common/exceptions.py [hunk 1]
```diff
@@ -70,36 +70,34 @@ def from_response(cls, resp, message=None):
...
-                b = '- %s' % self.http_reason
+                b = 'HTTP/S Response_Reason: %s\n' % self.http_reason
         if self.http_response_content:
-            if len(self.http_response_content) <= 60:
-                b += '   %s' % self.http_response_content
-            else:
-                b += '  [first 60 chars of response] %s' \
-                     % self.http_response_content[:60]
-        return b and '%s: %s' % (a, b) or a
+            b += 'Response_content: %s' % self.http_response_content
```
Comment (padthaitofuhot) [inline ecsclient/common/exceptions.py]: Recommend re-adding size conditional, but increasing size to something larger, such as 8k.

Comment (rm75) [reply ecsclient/common/exceptions.py]: Added 8K limit
https://github.com/EMCECS/python-ecsclient/pull/63/commits/a36e31af0fb5d07055bfa67519e1a83652c2771d

**Pull request merged**